pmlc/rt/vulkan/vulkan_runtime.cc

```diff
@@ -946,6 +999,16 @@ LogicalResult VulkanRuntime::createSchedule() {
     }
   }
 
+  /*
+  void vkCmdWriteTimestamp(
+    VkCommandBuffer                             commandBuffer,
+    VkPipelineStageFlagBits                     pipelineStage,
+    VkQueryPool                                 queryPool,
+    uint32_t                                    query);
+  */
+  vkCmdWriteTimestamp(commandBuffer, VK_PIPELINE_STAGE_BOTTOM_OF_PIPE_BIT,
```

Comment:
I am thinking about adding timestamps for each action (run a shader or a buffer transfer), what do you think?

Reply:
Good idea.  Will add this.

Reply:
Plan to add this in follow-up commit.